cmd/arc_summary/Makefile.am

```diff
@@ -1 +1,2 @@
 dist_bin_SCRIPTS = arc_summary.py
+dist_bin_SCRIPTS = arc_summary3.py
```

Comment:
You need to append `arc_summary3.py` to `dist_bin_SCRIPTS` otherwise `arc_summary.py` won't be included in the package which is why the test suite can find it and those tests are failing.

```
dist_bin_SCRIPTS = arc_summary.py arc_summary3.py
```